src/Program.cs

```diff
@@ -32,7 +32,7 @@ public static void Main(string[] args)
                 var xmlKey = File.ReadAllText("PublicKey.xml");
                 var rsa = new RSACryptoServiceProvider();
                 rsa.FromXmlString(xmlKey);
-                return rsa.VerifyData(computedSignature, archive.Signature, HashAlgorithmName.SHA1, RSASignaturePadding.Pkcs1);
+                return rsa.VerifyData(computedSignature, archive.Signature, HashAlgorithmName.SHA256, RSASignaturePadding.Pkcs1);
```

Comment:
This needs to be done based on metadata. Otherwise the checker will stop working for all existing archives.